src/main/java/com/iexec/common/result/ComputedFile.java

```diff
@@ -0,0 +1,32 @@
+package com.iexec.common.result;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ComputedFile {
+
+    @JsonProperty("deterministic-output")
```

Comment:
Sure, but developer are going to write it down pretty often, so we need to decide between short & clear

Reply:
Good point, personally I'd choose clearness since it might cause execution to fail in case the developer doesn't understand the meaning of the attribute.

Reply:
Updated, thanks